app/partials/content.html

```diff
@@ -1,15 +1,65 @@
-<div class="row marketing">
-  <div class="col-lg-6">
-    <h4>HTML5 Boilerplate</h4>
-    <p>HTML5 Boilerplate is a professional front-end template for building fast, robust, and adaptable web apps or sites.</p>
-
-    <h4>Sass</h4>
-    <p>Sass is the most mature, stable, and powerful professional grade CSS extension language in the world.</p>
+<br><br>
```

Comment:
Please try to avoid using `<br>`.
For spacing use margin or padding depending on the situation.